java/src/by/gsu/epamlab/task2/Material.java

```diff
@@ -0,0 +1,27 @@
+package by.gsu.epamlab.task2;
+
+public final class Material {
+    private String name;
+    private double density;
```

Comment:
The assignment requires the use of a constant for the variable. In turn, the constant requires mandatory initialization when creating an instance of the class with non-arg constructor.